PLUB/Sources/Views/Meeting/CreateMeeting/List/SelectPeopleNumberViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SelectPeopleNumberViewController.swift
+//  PLUB
+//
+//  Created by 김수빈 on 2023/01/09.
+//
+
+import UIKit
+
+final class SelectPeopleNumberViewController: BaseViewController {
+  
+  private var superView = UIViewController()
+  
+  // MARK: - Life Cycle
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+  }
+  
+  // MARK: - Configuration
+  
+  override func setupLayouts() {
+    super.setupLayouts()
+  }
+  
+  override func setupConstraints() {
+    super.setupConstraints()
+  }
+  
+  override func setupStyles() {
+    super.setupStyles()
+    view.backgroundColor = .red
+  }
+  
+  override func bind() {
+    super.bind()
+  }
+  
+  func setupSuperView(superView: UIViewController) {
+      self.superView = superView
+  }
```

Comment:
List 폴더에 있는 ViewController 3 곳에서 해당 코드만 4칸 띄워져있습니다. <kbd>⌃</kbd> + <kbd>I</kbd>를 이용해주시면 감사하겠습니다.

그리고 해당 PR에서는 이 코드가 실행되지 않는데 나중에 사용하기 위해 넣으신 코드인가요?

```suggestion
  func setupSuperView(superView: UIViewController) {
    self.superView = superView
  }
```

Reply:
앗 이부분은 이전에 설계했을 때의 코드인데 지워놓도록 하겠습니다!